fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/NumericAggregationValue.java

```diff
@@ -591,7 +677,53 @@ public enum PhysicalOperator {
         MAX_I(LogicalOperator.MAX, TypeCode.INT, TypeCode.INT, Objects::requireNonNull, (s, v) -> Math.max((int)s, (int)v), identity()),
         MAX_L(LogicalOperator.MAX, TypeCode.LONG, TypeCode.LONG, Objects::requireNonNull, (s, v) -> Math.max((long)s, (long)v), identity()),
         MAX_F(LogicalOperator.MAX, TypeCode.FLOAT, TypeCode.FLOAT, Objects::requireNonNull, (s, v) -> Math.max((float)s, (float)v), identity()),
-        MAX_D(LogicalOperator.MAX, TypeCode.DOUBLE, TypeCode.DOUBLE, Objects::requireNonNull, (s, v) -> Math.max((double)s, (double)v), identity());
+        MAX_D(LogicalOperator.MAX, TypeCode.DOUBLE, TypeCode.DOUBLE, Objects::requireNonNull, (s, v) -> Math.max((double)s, (double)v), identity()),
+        BITMAP_L(LogicalOperator.BITMAP, TypeCode.LONG, TypeCode.BYTES,
+                s -> {
+                    BitSet sset = new BitSet();
+                    sset.set(((Long)s).intValue());
+                    return sset;
+                },
+                (s, v) -> {
+                    BitSet sset = (BitSet)s;
+                    sset.or((BitSet)v);
+                    return sset;
+                },
+                s -> {
+                    int fixedResultArraySize = BitmapValueIndexMaintainer.DEFAULT_ENTRY_SIZE / 8;
+                    byte[] res = ((BitSet)s).toByteArray();
+                    if (res.length > BitmapValueIndexMaintainer.MAX_ENTRY_SIZE / 8) {
+                        throw new RecordCoreArgumentException("entry size option is too large")
+                                .addLogInfo("entrySize", res.length * 8, "maxEntrySize", BitmapValueIndexMaintainer.MAX_ENTRY_SIZE);
+                    } else if (res.length > fixedResultArraySize) {
+                        return res;
+                    } else {
+                        return Arrays.copyOf(res, fixedResultArraySize);
```

Comment:
```suggestion
                        return res;
```

reason: I guess this can be inefficient, because it will create a `10000` bitmap for _each_ element in the group, OTOH I am not 100% sure this works, so you might want to debug it.